README.md

```diff
@@ -1 +1,14 @@
-JMS4s Request-Reply Stub
\ No newline at end of file
+# JMS4s Request-Reply Stub
```

Comment:
Could we add some instructions for demonstrating this, using the restful API?

See [this comment](https://national-archives.atlassian.net/browse/PACT-709?focusedCommentId=39900).